### PR TITLE
Collector wasm tx

### PR DESCRIPTION
**Pull request by hanjukim**
## Description
This PR enables collector to index wasm msgs which is MsgStoreCode, MsgInstantiateContract, MsgExecuteContract, MsgMigrateContract, and MsgUpdateContractOwner by wallet address.

### CODE REVIEW

File: src/service/transaction/getTxList.ts [hunk 1]
```diff
@@ -116,30 +117,36 @@ async function getTxTotalCount(data: GetTxListParam): Promise<number> {
...
 }
 
 export async function getTxFromAccount(data: GetTxListParam, parse: boolean): Promise<GetTxsReturn> {
   if (!data.account) {
-    throw new Error(`Account address is required.`)
+    throw new TypeError(`Account address is required.`)
   }
 
-  const totalCnt = await getTxTotalCount(data)
+  if (!parseInt(data.limit as any, 10)) {
```
Comment (maruf0011) [inline src/service/transaction/getTxList.ts:134]: do we actually need these? as page and limit already validated by Joi.

File: src/service/transaction/getTxList.ts [hunk 2]
```diff
@@ -150,11 +157,16 @@ export async function getTxFromAccount(data: GetTxListParam, parse: boolean): Pr
...
+  const order: 'ASC' | 'DESC' = data.order && data.order.toUpperCase() === 'ASC' ? 'ASC' : 'DESC'
+  const orderAndPageClause = ` ORDER BY tx_id ${order} OFFSET ${offset} LIMIT ${Math.max(0, data.limit)}`
+
   const subQuery = `SELECT tx_id FROM (${distinctTxQuery}${orderAndPageClause}) a `
 
-  const query = `SELECT data, chain_id FROM tx WHERE id IN (${subQuery}) ORDER BY data->'timestamp' ${order}`
+  const query = `SELECT data, chain_id FROM tx WHERE id IN (${subQuery}) ORDER BY timestamp ${order}`
 
-  const txs = await getConnection().query(query)
+  console.log(query, params)
```
Comment (maruf0011) [inline src/service/transaction/getTxList.ts:168]: console.log

File: src/collector/getAddressFromMsg.ts [hunk 3]
```diff
@@ -35,54 +35,81 @@ export default function getAddressFromMsg(msg): { [key: string]: string[] } {
...
 
     case 'gov/MsgSubmitProposal': {
       return {
-        governance: get(msg, 'value.proposer') ? [get(msg, 'value.proposer')] : []
+        governance: [get(msg, 'value.proposer')].filter(Boolean)
       }
     }
 
+    case 'wasm/MsgStoreCode':
+      console.log('MsgStoreCode', msg)
```
Comment (maruf0011) [inline src/collector/getAddressFromMsg.ts:87]: console.log

File: src/service/transaction/getTxList.ts (hunk 2 above)
Comment (hanjukim) [reply src/service/transaction/getTxList.ts:168]: Thanks

File: src/collector/getAddressFromMsg.ts (hunk 3 above)
Comment (hanjukim) [reply src/collector/getAddressFromMsg.ts:87]: Thanks

File: src/service/transaction/getTxList.ts (hunk 1 above)
Comment (hanjukim) [reply src/service/transaction/getTxList.ts:134]: Yes I am aware of that but to be more safe..

**Review by maruf0011 — APPROVED** [review]

**Pull request merged**